drivers/sensor/tdk/icm456xx/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+zephyr_library()
+
+zephyr_library_sources(icm456xx.c)
+
+zephyr_library_sources_ifdef(CONFIG_SPI icm456xx_spi.c)
+zephyr_library_sources_ifdef(CONFIG_I2C icm456xx_i2c.c)
+zephyr_library_sources_ifdef(CONFIG_ICM456XX_TRIGGER icm456xx_trigger.c)
+zephyr_library_sources_ifdef(CONFIG_TDK_APEX icm456xx_apex.c)
```

Comment:
We can see that icm45686.c source file is not used?

This file icm45686.c should not exist, I understand you may keep the files organization from icm42x70 folder but as we support icm45686 which is a standard 6-axis part variant, we can only have an icm456xx.c.

For your information, in icm42x70 folder, the icm42670.c file was only needed to build for 6-axis variant and excluded for icm42370 3-axis.

Reply:
Yes I will remove it